e2e/payment/cancel.spec.ts

```diff
@@ -19,7 +21,7 @@ describe('IT, cancel', () => {
 					]
 				},
 				payer: {
-					email: 'test_user_123@testuser.com',
+					email: email,
```

Comment:
Quando a o nome da chave de um objeto for o mesmo do nome da variável que esta atribuindo, por exemplo 
`email: email,` nesse caso podemos omitir o nome da variável, ficando apenas `email,` 

Ou se preferir, não salve o retorno dessa função em uma variável para depois atribuir ao objeto, vc pode atribuir o valor do retorno diretamente
`email: createEmailTestUser(),`